src/components/ListOrgs.vue

```diff
@@ -272,6 +276,70 @@ const exportAll = async () => {
   exportCsv(exportData, `roar-${activeOrgType.value}.csv`);
 };
 
+const exportOrgUsers = async (orgType) => {
```

Comment:
Not going to block the PR for this, but two quick comments:

* This is the type of function that would best live outside of the component in form of service. But this would require some additional tweaks. Let me know if this is something you want to explore and I'd be happy to elaborate. 

* Not sure if this is something the team wants to do, but personally I'd add a JSDoc comment here, mostly to document the approach and decision here. Something along the lines of: 

```suggestion
/**
 * Exports users of a given organization type to a CSV file.
 * 
 * @NOTE In order to avoid overly large exports, the function will allow exports up to a predefined limit (currently
 * 10,000 records). To avoid running a large and potentially unecessary query, we first run an aggregation query to
 * verify that the export is within the limit.
 * 
 * @TODO Replace this logic with a server driven export, for example a cloud function that generate a download link for
 * the user, effectively allowing complete and large exports.
 * 
 * @param {Object} orgType - The organization type object.
 * @param {string} orgType.id - The ID of the organization type.
 * @param {string} orgType.name - The name of the organization type.
 * 
 * @returns {Promise<void>} - A promise that resolves when the export is complete.
 */
const exportOrgUsers = async (orgType) => {
```